src/core/signature.zig

```diff
@@ -1,43 +1,64 @@
 const std = @import("std");
-const Pubkey = @import("pubkey.zig").Pubkey;
+const sig = @import("../sig.zig");
+const core = @import("lib.zig");
+
 const Ed25519 = std.crypto.sign.Ed25519;
 const Verifier = std.crypto.sign.Ed25519.Verifier;
 const e = std.crypto.errors;
 
-pub const SIGNATURE_LENGTH: usize = 64;
+const Pubkey = core.Pubkey;
+
+pub const SIGNATURE_LENGTH: usize = Signature.size;
```

Comment:
Ditto here, why not just use `Signature.size`?

Reply:
same as above. I'm not opposed to changing it, just didn't feel in scope at the time.

Reply:
ef6543db2bce46db98fefc69cddcfc48d37d4439